c/dataservice.c

```diff
@@ -297,8 +297,10 @@ WebPlugin *makeWebPlugin(char *pluginLocation, JsonObject *pluginDefintion, Inte
       }
     }
 
-    printf("For plugin=%s, found %d data service(s)\n", plugin->identifier, plugin->dataServiceCount);
-    plugin->dataServices = (DataService**)safeMalloc(sizeof(DataService*) * plugin->dataServiceCount,"DataServices");
+    zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_INFO, "Plugin=%s, found %d data service(s)\n", plugin->identifier, plugin->dataServiceCount);
+    if (plugin->dataServiceCount > 0) {
+      plugin->dataServices = (DataService**)safeMalloc(sizeof(DataService*) * plugin->dataServiceCount,"DataServices");
```

Comment:
@1000TurquoisePogs are we sure that the `dataServices` field is initialized and if not, that it's safe not to initialize it to NULL? I'm asking because that safeMalloc would assign NULL in the current version.

Reply:
No I'm not sure. Maybe, it is better to else{plugin->dataServices=NULL} for consistency.